src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -45,7 +45,7 @@
                       </div>
                     </template>
                     <span>The updated office addresses will be legally effective on {{filing.effectiveDate}},
-                      12:01 AM (Pacific Time). No other filings are allowed until then.</span>
+                      12:01 am Pacific time. No other filings are allowed until then.</span>
```

Comment:
This and changes below are based on BC Gov web style guide:

https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/web-content-development-guides/web-style-guide/writing-guide/numbers